HopsanGUI/Dialogs/OptimizationScriptWizard.cpp

```diff
@@ -846,6 +869,7 @@ void OptimizationScriptWizard::addFunction()
 {
     int idx = mpFunctionsComboBox->currentIndex();
     addObjectiveFunction(idx, 1.0, 1.0, 2.0, mSelectedVariables, QStringList());
+    emit(currentPage()->completeChanged());
```

Comment:
This fail to build with Qt4 since the signal is protected. Maybe it can be emitted directly from the new page subclass somehow.
It might also be possible to chain link a signal from the wizard to the signal inside the page. Signal to signal connection.
In that case the connection must be made inside the page class and you need a new signal in the wizard class